Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -3056,7 +3057,7 @@ Required license: Cortex XDR Pro per Endpoint, Cortex XDR Pro, or Cortex XDR Pro
 
 ***
 Retrieve a list of the current user emails associated with one or more user groups in the environment.
-
+Required license: Cortex XDR Pro per Endpoint, Cortex XDR Pro, or Cortex XDR Pro per GB.
```

Comment:
Please enforce the new line, You can see in the preview one line break is not shown.
https://stackoverflow.com/questions/36583502/how-to-force-a-linebreak